### ISSUE TITLE
[Rogue] Killing Spree - Combat Potency proc

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#3994]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While Killing Spree is going on Combat Potency does not proc.

**Expected behaviour**: Tell us what should happen instead
Combat Potency should proc on both the 5 successful attacks from the off-hand during the duration and from the extra melee attacks done during. 

**Steps to reproduce the problem**:

1. Find an appropriate target
2. Use Killing Spree
3. See that you are not getting Energy

**Include proofs for this behaviour**
http://www.wowhead.com/spell=51690/killing-spree#comments - 7th comment. There really isn't a lot of talk about this because it is not the skill's main objective. But getting energy back while using it is certainly a bonus.

Spell ID = 51690

**Comment by JarlBallin** [comment]
@Rushor

**Comment by Rushor** [comment]
Patch 3.0.3 (04-Nov-2008): Now only works with auto attacks (no more Shiv.)

So this works blizzlike. If you find a propper source that Killing Spree should proc Combat Potency on 3.3.5 i will reopen the issue. On Hold for now.

**Comment by InshadowsAlgalon** [comment]
"KS should be used when you are low on energy, because during it your energy will surely regenerate to full - Combat Potency can proc from KS." - Warmane Cata Rogue Guide http://forum.warmane.com/showthread.php?t=176141

"One important thing to know is that KS, along with its high burst damage, helps you regenerate 
your energy. It costs no energy to start with, while casting you won't be using any abilities so your 
energy will refill (Vitality talent), and the guaranteed off hand attacks with "combat potency" talent 
will fill out your energy pool. As a consequence to this, it is important to use KS only after you 
spend all your energy. "
https://archive.org/stream/CombatRogueGuide/combat%20rogue%20guide______swordz_djvu.txt

https://www.youtube.com/watch?v=LYe_1qmCeRM   on 1:25 he pops KS and gets 2x15 energy during it
on 4:13 he get 15 eergy one time.

Both sources are from Warmane-Molten ( one is from Cata), couldnt find more difinitive proof, nor did i find any statement which confirms that you CANT get combat potency from KS.
And from a video you cant 100% tell that its from KS itself and not from melee hits. But u have to admit that its somewhat suspicious that he gets those combat potency procs, given that he is in pvp spec and environment (without buffs, especially haste buffs which would mean more melee hits while in KS).

**Comment by TeridanDW** [comment]
Hello @InshadowsAlgalon , I am sorry but i had to edit part of your post as it contained links to diffrent private server and what is said there can't be used a proof as it's not a valid source.

> propper source

is only retail (3.0.3 - 3.3.5) videos, wowhead comments (pinged onces), wowwiki etc.

**Comment by lost-illusion** [comment]
@Rushor 
https://www.youtube.com/watch?v=LYe_1qmCeRM 1:23 
Correct ?

**Comment by mercred** [comment]
Taking WOL from retail dated from june 2010 -http://www.worldoflogs.com/reports/rt-36ctwg9o5bmntva8/ if you run an expression on saurfang fight such as  **sourceName = "Kozak" AND (type = TYPE_GAIN
OR ( spell = "Killing Spree" AND (type = TYPE_CAST OR type=TYPE_DAMAGE)))**
you will see the following picture:
![Combat potency during KS](https://user-images.githubusercontent.com/15626206/59496736-6311ef80-8e92-11e9-816c-d943d98e9fcc.png)
Hope this can be considered a valid source,please re-open
